components/sidebar/sidebar_channel/sidebar_channel_menu/sidebar_channel_menu.tsx

```diff
@@ -209,32 +209,16 @@ export class SidebarChannelMenu extends React.PureComponent<Props, State> {
 
         let leaveChannel;
         if (channel.name !== Constants.DEFAULT_CHANNEL) {
-            if (channel.type === Constants.DM_CHANNEL || channel.type === Constants.GM_CHANNEL) {
-                leaveChannel =
+            leaveChannel =
```

Comment:
we typically leave the opening `(` in the same line with the assignment

```suggestion
            leaveChannel = (
```

I know that we have been inconsistent with this historically, so could you please add this change in as well? Thanks!

Reply:
No worries